src/containers/PatientProfilePage.tsx

```diff
@@ -94,17 +94,17 @@ const PatientProfilePage = ({
   }, [data]);
```

Comment:
```suggestion
  }, [data, formFields, loading, mode]);
```

Reply:
i tested this again on an edit patient page (http://localhost:3000/patients/edit/3/1) and i get into an infinite loop with the page going unresponsive
![image](https://user-images.githubusercontent.com/59516223/87887876-f226fa80-c9f6-11ea-8b3d-acf6405f9c5e.png)

Reply:
moved this into a separate ticket - needs to be fixed before launch
https://www.notion.so/uwblueprintexecs/useEffect-error-7108d3eee3cf4c96893e3de358ae5a55